stats/albatross_stats_stubs.c

```diff
@@ -135,7 +135,13 @@ CAMLprim value vmmanage_vmmapi_stats (value octx) {
   const char *desc;
   struct vmctx *ctx = (struct vmctx*)octx;
 
+#if __FreeBSD_version >= 1400000
+  struct vcpu *vcpu;
+  vcpu = vm_vcpu_open(ctx, 0);
```

Comment:
`vm_vcpu_open` allocates memory, shouldn't `vm_vcpu_close` be called somewhere?

Reply:
indeed, thanks for the comment -- does [057463f](https://github.com/robur-coop/albatross/pull/172/commits/057463f0765483be09598be91c6cc547dac8d5f6) address your concerns?

Reply:
Yes thanks!